diff --git a/dist/esm/pack/index.js b/dist/esm/pack/index.js
index 97e262734c587f31b89aeaf370d11bb4760f33f0..fe0f34ce16a9a5a79174a0d9f7a33c6d63a9e637 100644
--- a/dist/esm/pack/index.js
+++ b/dist/esm/pack/index.js
@@ -13,6 +13,7 @@ export async function pack({ input, blockstore: userBlockstore, hasher, maxChunk
     // Consume the source
     const rootEntry = await last(pipe(getNormaliser(input), (source) => importer(source, blockstore, {
         ...unixfsImporterOptionsDefault,
+        cidVersion: 0,
         hasher: hasher || unixfsImporterOptionsDefault.hasher,
         maxChunkSize: maxChunkSize || unixfsImporterOptionsDefault.maxChunkSize,
         maxChildrenPerNode: maxChildrenPerNode || unixfsImporterOptionsDefault.maxChildrenPerNode,
